Explorer/Assets/DCL_RenderFeature_ProceduralSkyBox.cs

```diff
@@ -0,0 +1,196 @@
+using System;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+using UnityEngine.Experimental.Rendering;
+
+[Serializable]
+internal class ProceduralSkyBoxSettings_Generate
+{
+    // Parameters
+    [SerializeField] internal int SunDisk = 2;
+    [SerializeField] internal float SunSize = 1.0f;
+    [SerializeField] internal float SunSizeConvergence = 5.0f;
+    [SerializeField] internal float AtmosphereThickness = 1.0f;
+    [SerializeField] internal Color GroundColor = new Color(0.369f, 0.349f, 0.341f, 1.0f);
+    [SerializeField] internal Color SkyTint = new Color(0.5f, 0.5f, 0.5f, 1.0f);
+    [SerializeField] internal float Exposure = 1.3f;
+    [SerializeField] internal Vector4 SunPos = new Vector4(-0.04f, -0.02f, 0.0f, 1.0f);
+    [SerializeField] internal Vector4 SunColour = new Vector4(1.0f, 1.0f, 1.0f, 1.0f);
+}
+
+[Serializable]
+internal class ProceduralSkyBoxSettings_Draw
+{
+    // Parameters
+
+}
+
+public partial class DCL_RenderFeature_ProceduralSkyBox : ScriptableRendererFeature
```

Comment:
Do you mind following our coding convention?
`Rider` offers a "silent clean up" shortcut, you should have "Code Style Reformal DCL" set for it
The same profile should be chosen as a git commit hook
<img width="341" alt="image" src="https://github.com/decentraland/unity-explorer/assets/118179774/6dd19764-3a10-4404-aae9-0a2face9e842">